x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -126,12 +126,18 @@ export const ActionForm = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const isCaseConnector = (connector: ActionConnector) =>
+    !connector.isPreconfigured &&
+    (connector?.config?.isCaseOwned === true || connector.config?.incidentConfiguration?.mapping);
```

Comment:
Isn't `connector?.config?.isCaseOwned === true` should be enough to check? Why do we hide `connector.config?.incidentConfiguration?.mapping`?

Reply:
The reason I used `connector.config?.incidentConfiguration?.mapping` is because `isCaseOwned` was introduced after `mapping` and some connectors created prior do not have the `isCaseOwned` property.